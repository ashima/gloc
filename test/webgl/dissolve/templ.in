#if 1
#define T(t) t fn(t a, t b) { return t(a.x+b.x); }

T(vec2)
T(vec3)
T(vec4)
#undef T
#endif
