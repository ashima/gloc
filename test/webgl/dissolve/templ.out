{
  "glo": [ 1, 0, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "units": [
    {
      "outsym": [ "fn" ],
      "source":
        "#line 1 GLOC_0\n#if 1\n#define T(t) t fn(t a, t b) { return t(a.x+b.x); }\n\nT(vec2)\nT(vec3)\nT(vec4)\n#undef T\n#endif"
    }
  ],
  "linkmap": { "0": "#n=0" }
}
